ui/DesignSystem/Component/TrackToggle.svelte

```diff
@@ -6,41 +6,147 @@
 
   export let style = null;
   export let peerCount = null;
+  let hover = false;
+  let active = false;
+  let tracking = false;
+  const enter = () => {
```

Comment:
Since these are simple one-liners and used only in one place you could as well inline them where they're used.
```
  on:mouseenter={() => { hover = true; }}
  on:mousedown={() => { active = true; }}
  on:mouseup={() => { active = false; tracking = !tracking;}}
  on:mouseleave={() => { hover = false; }}>
```

Reply:
my editor styles this automagically...

Reply:
lol, never mind then